tasks/main.yml

```diff
@@ -30,33 +31,69 @@
     state: "{{ state | default(omit) }}"
   when: "'notification' not in skip_tasks"
   tags:
-    - system_alarms
-    - monasca_alarms
-    - openstack_alarms
-    - service_alarms
+    - always
   register: default_notification
 
 # Include the various alarm sets
-- import_tasks: system.yml
+- include_tasks: alarms.yml
```

Comment:
I suppose you could loop over the different groups here, but no need to over-complicate it.

Reply:
That'd be nicer certainly. I couldn't figure out how to preserve the original tags inside a loop though.